lib/kitchen/provisioner/install.erb

```diff
@@ -8,11 +9,12 @@ salt_apt_repo_key = config[:salt_apt_repo_key]
 salt_ppa = config[:salt_ppa]
 
 <<-INSTALL
-sh -c '
+bash -c '
```

Comment:
Why not stick using to `sh`? `bash` is not installed on all environments, while `sh` will be symlinked to a `sh` compatible shell.